src/client/pydaos/raw/daos_api.py

```diff
@@ -461,15 +462,20 @@ def get_attr(self, attr_names, poh=None, cb_func=None):
         if poh is not None:
             self.handle = poh
 
+        buffer_size = 100
+        # Adjust the buffer size based on attributes to retrieve
+        for attr in attr_names:
+            if buffer_size <= len(attr):
+                buffer_size = len(attr)
         attr_count = len(attr_names)
         attr_names_c = (ctypes.c_char_p * attr_count)(*attr_names)
 
         no_of_att = ctypes.c_int(attr_count)
         buffers = ctypes.c_char_p * attr_count
-        buff = buffers(*[ctypes.c_char_p(ctypes.create_string_buffer(100).raw)
+        buff = buffers(*[ctypes.c_char_p(ctypes.create_string_buffer(buffer_size).raw)
```

Comment:
(pylint-line-too-long) Line too long (86/80)

Reply:
Edited line to be less than 80 chars long.